Example/metro.config.js

```diff
@@ -1,21 +1,17 @@
-const blacklist = require('metro-config/src/defaults/blacklist');
-const path = require('path');
-const cwd = path.resolve(__dirname);
-
-const glob = require('glob-to-regexp');
-
-function getBlacklist() {
-  const nodeModuleDirs = [
-    glob(`${process.platform === 'win32' ? path.resolve(__dirname, '..').replace(/\\/g, '/') : path.resolve(__dirname, '..')}/node_modules/*`),
-    glob(`${process.platform === 'win32' ? path.resolve(__dirname).replace(/\\/g, '/') : path.resolve(__dirname)}/node_modules/metro/node_modules/fbjs/*`),
-  ];
-  return blacklist(nodeModuleDirs);
-}
+/**
+ * Metro configuration for React Native
+ * https://github.com/facebook/react-native
+ *
+ * @format
+ */
 
 module.exports = {
-  resolver: {
-    blacklistRE: getBlacklist(),
-    providesModuleNodeModules: ['react-native', 'react', 'fbjs'],
+  transformer: {
```

Comment:
I'd prefer if we keep this config. It have us a way to import source from reanimated directly from top level directory which helps a lot with development

Reply:
I can revert the change, but it's not working either

Reply:
I think this setup no longer works with metro from 0.61 and @osdnk had some workaround for it. Lets revert for now or sync up on getting that workaround pulled in